dlinfer/vendor/maca/maca_ops.py

```diff
@@ -326,7 +326,7 @@ def moe_gating_topk_softmax(
 
 
 @register_ops(vendor_ops_registry)
-def silu_and_mul(x: Tensor) -> Tensor:
+def silu_and_mul(x: Tensor, dim: int = -1) -> Tensor:
```

Comment:
why need dim?

Reply:
to be compatible with the silu_and_mul interface in llm.py, this dim parameter was added at some commit;  the dim always uses the default value of -1 in maca backend.